tfidf-benchmark/NLP-pipeline-cuML.ipynb

```diff
@@ -0,0 +1,2955 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">&nbsp; &nbsp; if benchmark:
&nbsp; &nbsp; &nbsp; &nbsp; with dask.annotate(workers=set(client.has_what().keys())):
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; result = client.persist(result)
&nbsp; &nbsp; &nbsp; &nbsp; wait(result)
</pre><p> </p><p>We should add these four lines to a function as this is repeated across the workflow. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Another question ? 

```
 with dask.annotate(workers=set(client.has_what().keys())):
 ```
 
 Is this line needed as to me it seems that it will scale to all the workers that client has ?

Reply:
1. That makes sense. Would you prefer if I just did `client.persist()` an then `wait` instead of the dask annotations?
2. Actually It would not since I am scaling the number of workers to a fixed number before each run starts.